css/select_a_meal.css

```diff
@@ -44,7 +44,9 @@
     color:#4e4e4e;
 
 }
-
+.lab-selected{
+    background:#4e4e4e !important;
```

Comment:
There's GOTTA be a way to not use !important here :) If the background color is not getting shown, use a more specific query selector and you won't need to use important. If you don't care to make that change I'm not insulted though. !important is just considered bad practice.